src/Transport/Topology/TopologyCreator.cs

```diff
@@ -1,21 +1,41 @@
 namespace NServiceBus.Transport.AzureServiceBus
 {
+    using System;
     using System.Linq;
     using System.Threading.Tasks;
+    using Logging;
+    using NServiceBus.AzureServiceBus.Utils;
+    using Settings;
 
-    class TopologyCreator : ICreateTopology, ITearDownTopology
+    class TopologyCreator : ICreateTopologyInternal
     {
+        static ILog Logger = LogManager.GetLogger<TopologyCreator>();
         ITransportPartsContainer container;
         IManageNamespaceManagerLifeCycle namespaces;
+        AsyncLazy<bool> hasManageRights;
+        string namespacesWithoutManageRightsJoined;
 
         public TopologyCreator(ITransportPartsContainer container, IManageNamespaceManagerLifeCycle namespaces)
         {
             this.container = container;
             this.namespaces = namespaces;
+            hasManageRights = new AsyncLazy<bool>(async () =>
+            {
+                var namespacesWithoutManageRights = await ManageRightsCheck.Run(namespaces, container.Resolve<ReadOnlySettings>())
+                    .ConfigureAwait(false);
+                namespacesWithoutManageRightsJoined = string.Join(", ", namespacesWithoutManageRights.Select(alias => $"`{alias}`"));
+                return namespacesWithoutManageRights.Count == 0;
+            });
         }
 
         public async Task Create(TopologySection topology)
         {
+            if (!await hasManageRights)
+            {
+                Logger.Info($"Configured to create topology, but have no manage rights for the following namespace(s): {namespacesWithoutManageRightsJoined}. Execution will continue and assume the topology is already created.");
```

Comment:
Nope for that we have the AssertManagedRights method. The create will just silently bypass if the rights are not given